### PR TITLE
try again

### PR DESCRIPTION
**Pull request by k4leung4**
this should trigger release when merged
needs to be under charts/

### CODE REVIEW

File: charts/enforce-agent/Chart.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ description: |
 
   Documentation for the chart can be found [here](https://edu.chainguard.dev/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/alternative-installation-methods/).
 type: application
-version: 0.0.29
+version: 0.0.39
```
Comment (vaikas) [inline charts/enforce-agent/Chart.yaml]: is this intended? Or .30?

Comment (k4leung4) [reply charts/enforce-agent/Chart.yaml]: doh, fixed

File: charts/enforce-agent/README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 <!-- This README.md is generated. Please edit README.md.gotmpl -->
 
-![Version: 0.0.25](https://img.shields.io/badge/Version-0.0.25-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v0.1.96](https://img.shields.io/badge/AppVersion-v0.1.96-informational?style=flat-square)
+![Version: 0.0.30](https://img.shields.io/badge/Version-0.0.25-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v0.1.100](https://img.shields.io/badge/AppVersion-v0.1.100-informational?style=flat-square)
```
Comment (vaikas) [inline charts/enforce-agent/README.md]: URL from 0.0.25 => 0.0.30 also?

Comment (k4leung4) [reply charts/enforce-agent/README.md]: yes, hasnt been updated for a long time

Comment (vaikas) [reply charts/enforce-agent/README.md]: Sorry, meant this part of the actual URL:
`https://img.shields.io/badge/Version-0.0.25-informational?` should that be 0.0.30 as well?

Comment (k4leung4) [reply charts/enforce-agent/README.md]: good catch, fixed

**Review by vaikas — APPROVED** [review]

**Pull request merged**